.github/workflows/ci.yaml

```
@@ -1,32 +1,43 @@
 on:
   repository_dispatch:
-    types: [spec_release]
+    types: [release]
   workflow_dispatch:
-
-#on: push
+    inputs:
+      json:
+        description: 'Passed json from repository_dispatch'
+        required: true
+        type: string
+      version_postfix:
+        description: 'Additional string to concatenate onto the existing version before release'
+        required: false
+        type: string
+        default: ''
 
 name: Generate VRChat API SDK
 
 jobs:
   generate:
     runs-on: ubuntu-latest
     name: Generate VRChat API SDK
+    env:
+      ARTIFACT_NAME: "openapi.yaml"
+      INPUT: ${{ github.event_name == 'repository_dispatch' && github.event.client_payload || inputs.json }}
+      SPEC_URL: ${{ fromJSON(github.event_name == 'repository_dispatch' && github.event.client_payload || inputs.json)['artifacts']['openapi.yaml'] }}
+      PASSED_VERSION: ${{ fromJSON(github.event_name == 'repository_dispatch' && github.event.client_payload || inputs.json)['version'] }}
```

```suggestion
      INPUT: ${{ github.event_name == 'repository_dispatch' && toJSON(github.event.client_payload) || inputs.json }}
      SPEC_URL: ${{ (github.event_name == 'repository_dispatch' && github.event.client_payload || fromJSON(inputs.json))['artifacts']['openapi-legacy.yaml'] }}
      PASSED_VERSION: ${{ (github.event_name == 'repository_dispatch' && github.event.client_payload || fromJSON(inputs.json))['version'] }}
```

The `github.event.client_payload` is already an Object, but `inputs.json` is a string.

I found this out, after Rust received it's first repository dispatch